### PR TITLE
Prevent test binary from deploying

### PR DESCRIPTION
**Pull request by joshuajbouw**
Just noticed a dangerous situation. We have a new public function called `mint_account` for `integration-test` feature which allows you to make any address, at any nonce, at any balance. If someone accidentally built a test build and uploaded that, it would allow anyone to use that method.

To address that, I just changed the .wasm  test file compilations specifically state that they are test binaries and looking into adding something deploy that causes it to explode if someone accidentally deploys the test version.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -189,7 +189,14 @@ mod contract {
     /// MUTATIVE METHODS
     ///
 
+    #[cfg(feature = "integration-test")]
+    #[no_mangle]
+    pub extern "C" fn deploy_code() {
+        sdk::panic_utf8(b"ERR_DEPLOYED_TEST_VERSION");
```
[the remark this answers is not in the document]
Comment (joshuajbouw) [reply src/lib.rs:195]: hm

[the remark this answers is not in the document]
Comment (sept-en) [reply src/lib.rs:195]: I agree. I think this thing needs to be reverted ASAP

[the remark this answers is not in the document]
Comment (sept-en) [reply src/lib.rs:195]: @joshuajbouw

[the remark this answers is not in the document]
Comment (sept-en) [reply src/lib.rs:195]: Fixed in #235 (c815108dd commit)

**Pull request merged**